tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/Transports/LargePayload/LargePayloadTestBase.cs

```diff
@@ -0,0 +1,109 @@
+// <copyright file="LargePayloadTestBase.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using Datadog.Trace.TestHelpers;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Datadog.Trace.ClrProfiler.IntegrationTests
+{
+    public class LargePayloadTestBase : TestHelper
+    {
+        public LargePayloadTestBase(ITestOutputHelper output)
+            : base("LargePayload", @"test\test-applications\regression", output, prependSamplesToAppName: false)
+        {
+        }
+
+        public int FillerTagLength => 100;
+
+        public int TracesToTrigger => 100;
+
+        public int SpansPerTrace => 1999;
+
+        public int ExpectedSpans => TracesToTrigger + (TracesToTrigger * SpansPerTrace);
+
+        protected void RunTest()
+        {
+            using (var agent = EnvironmentHelper.GetMockAgent())
+            {
+                using (var sample = RunSampleAndWaitForExit(agent, arguments: $" -t {TracesToTrigger} -s {SpansPerTrace} -f {FillerTagLength}"))
+                {
+                    var spans = agent.WaitForSpans(ExpectedSpans);
+                    AssertLargePayloadExpectations(spans);
+                }
+            }
+        }
+
+        private void AssertLargePayloadExpectations(IImmutableList<MockTracerAgent.Span> spans)
```

Comment:
Think this will be broken when you rebase
```suggestion
        private void AssertLargePayloadExpectations(IImmutableList<MockSpan> spans)
```